mijia/src/lib.rs

```diff
@@ -25,9 +28,16 @@ const CONNECTION_INTERVAL_CHARACTERISTIC_PATH: &str = "/service0021/char0045";
 const CLOCK_CHARACTERISTIC_PATH: &str = "/service0021/char0022";
 const TEMPERATURE_UNIT_CHARACTERISTIC_PATH: &str = "/service0021/char0032";
 const COMFORT_LEVEL_CHARACTERISTIC_PATH: &str = "/service0021/char0042";
+const HISTORY_RANGE_CHARACTERISTIC_PATH: &str = "/service0021/char0025";
+const HISTORY_DELETE_CHARACTERISTIC_PATH: &str = "/service0021/char003f";
+const HISTORY_LAST_RECORD_CHARACTERISTIC_PATH: &str = "/service0021/char002b";
+const HISTORY_INDEX_CHARACTERISTIC_PATH: &str = "/service0021/char0028";
+const HISTORY_RECORDS_CHARACTERISTIC_PATH: &str = "/service0021/char002e";
```

Comment:
what does this look like if it is sorted by path?

Reply:
As good as any other way. Done.